controllers/platform/k8s.go

```diff
@@ -233,8 +233,12 @@ func createOrUpdateService(ctx context.Context, client client.Client, platform *
 
 func getLabels(platform *operatorapi.SonataFlowPlatform, psh services.PlatformServiceHandler) (map[string]string, map[string]string) {
 	lbl := map[string]string{
-		workflowproj.LabelApp:     platform.Name,
-		workflowproj.LabelService: psh.GetServiceName(),
+		workflowproj.LabelApp:          platform.Name,
+		workflowproj.LabelService:      psh.GetServiceName(),
+		"app.kubernetes.io/name":       psh.GetServiceName(),
+		"app.kubernetes.io/component":  psh.GetContainerName(),
+		"app.kubernetes.io/part-of":    platform.Name,
+		"app.kubernetes.io/managed-by": "sonataflow-operator",
```

Comment:
Please add these to https://github.com/apache/incubator-kie-kogito-serverless-operator/blob/main/api/metadata/annotations.go#L22